diff -urN gmpc-serverstats-0.15.5.0/src/Makefile.am gmpc-serverstats-0.15.5.0.new/src/Makefile.am
--- gmpc-serverstats-0.15.5.0/src/Makefile.am	2007-11-29 14:57:22.000000000 +0100
+++ gmpc-serverstats-0.15.5.0.new/src/Makefile.am	2008-02-04 11:00:23.179821052 +0100
@@ -1,6 +1,6 @@
-libdir=${prefix}/share/gmpc/plugins/
+pluginsdir=$(libdir)/gmpc/
 
-lib_LTLIBRARIES = serverstatsplugin.la
+plugins_LTLIBRARIES = serverstatsplugin.la
 
 serverstatsplugin_la_SOURCES=	\
 	plugin.c 
